pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/OmsorgEgenerklaeringFraser.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.pensjon.brev.maler.fraser
+
+import no.nav.pensjon.brev.maler.fraser.common.Felles
+import no.nav.pensjon.brev.template.Expression
+import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.Language.English
+import no.nav.pensjon.brev.template.Language.Nynorsk
+import no.nav.pensjon.brev.template.LanguageSupport
+import no.nav.pensjon.brev.template.dsl.TemplateRootScope
+import no.nav.pensjon.brev.template.dsl.expression.expr
+import no.nav.pensjon.brev.template.dsl.expression.plus
+import no.nav.pensjon.brev.template.dsl.text
+import no.nav.pensjon.brev.template.dsl.textExpr
+
+fun TemplateRootScope<LanguageSupport.Triple<Bokmal, Nynorsk, English>, *>.omsorgEgenerklaeringTittel() {
```

Comment:
Hmm, grunnen til at jeg foreslo to fraser var fordi jeg tenkte på `object : ...Phrase` som ikke er mulig i RootScope, og man dermed måtte ha:
```
title {
    includePhrase(OmsorgEgenerklaeringTittel)
}
outline {
   includePhrase(OmsorgEgenerklaeringOutline)
}
```
Jeg tenkte ikke på å lage extension funksjon, så da kan du også vurdere om du bare skal gå for å ha _en_ funksjon?

Reply:
Etter å ha tenkt litt på det trur eg denne varianten er betre, så gjekk over til det. Det er noko med å introdusere så få nye mønstre som mogleg.